### DISCUSSION TITLE
🙌 创建了一个 Formily 的 GPT，叫 Formily 专家，供各位答疑解惑

### DISCUSSION THREAD

**Discussion by jas0ncn** [alibaba/formily#4031] [Show and tell]
链接：https://chat.openai.com/g/g-opIdx6v9N-formily-zhuan-jia

NOTE：需要开通 ChatGPT Plus

**Comment by esmadrider** [comment]
很不错，立马用上了，请问是怎么训练的呀？喂养 PDF 还是喂养源码还是？

**Reply by jas0ncn** [reply]
把 formily 的文档都 down 下来，合并之后喂给 GPT

**Reply by Hew007** [reply]
请问大佬，有相关的博客或者文章吗，学习参考下，非常感谢！
